### PR TITLE
taxonomy: correction sweet spreads

### PR DESCRIPTION
**Pull request by moon-rabbitOFF**
### Related issue(s) and discussion

- Fixes #[ISSUE NUMBER]

### CODE REVIEW

File: taxonomies/categories.txt [hunk 1]
```diff
@@ -93462,10 +93491,6 @@ pt:Doces de figo
 sv:Fikonsylter
 wikidata:en:Q65545224
 
-<en:Jams
-en:ginger jams
-fr:Confitures de gingembre
-
```
Comment (benbenben2) [inline taxonomies/categories.txt:93468]: Is it possible to keep separated "jams" vs "preserves" for ginger?

The reason is that ginger jams in EU should have 150g of fruit per 1000g. Which is not the case for preserves. (see: https://github.com/openfoodfacts/openfoodfacts-server/pull/9707/files and https://eur-lex.europa.eu/legal-content/EN/TXT/?uri=CELEX%3A02001L0113-20131118)

Comment (moon-rabbitOFF) [reply taxonomies/categories.txt:93468]: Ah yes, didn't see that sorry. Should then put ginger jams under ginger preserves?

Comment (benbenben2) [reply taxonomies/categories.txt:93468]: Yes, that should be fine like this.

**Review by benbenben2 — APPROVED** [review]
Super. Thanks @moon-rabbitOFF for the correction.

**Pull request merged**